### PR TITLE
daily check for component graph updates

### PR DESCRIPTION
**Pull request by tiraboschi**
daily check for component graph updates,
open a PR only if we have a change in the source
files (*.gv).

Signed-off-by: Simone Tiraboschi <stirabos@redhat.com>

**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly
  
**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by tiraboschi** [comment]
false positive on coverage
/override coverage/coveralls

File: .github/workflows/graphs-updater.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 name: Update component graphs from the nightly job
-on: workflow_dispatch
+on:
+  schedule:
+    - cron:  '0 8 * * *'
+  workflow_dispatch:
 jobs:
   build:
     name: Update Dashboard ConfigMaps
```
Comment (orenc1) [inline .github/workflows/graphs-updater.yml:8]:
```suggestion
    name: Update Components Graph
```

Comment (tiraboschi) [reply .github/workflows/graphs-updater.yml]: done, thanks

**Review by orenc1 — APPROVED** [review]
/approve

**Pull request merged**